src/script/cpp_api/s_base.h

```diff
@@ -60,13 +60,15 @@ extern "C" {
 enum class ScriptingType: u8 {
 	Async,
 	Client,
+	CSM, // TODO: Remove distinction between this and Client
```

Comment:
We can't or shouldn't merge cpcsm and sscsm into one. They are too different, i.e. sscsm has features that would allow cheating.

Btw. `CPCSM` and `SSCSM` would be easier to understand and distinct names than `Client` and `CSM`.

Reply:
It would be possible to change the behavior of e.g. `get_node` depending on whether the CSMs are server-sent or client-provided. The intent is to ease the maintenance burden and keep CPCSMs from being neglected. However, I am inclined to agree with you, especially since executing code in a separate process is a lot slower. Perhaps I should add this to the agenda of the next meeting.